CMakeLists.txt

```diff
@@ -124,7 +124,8 @@ add_subdirectory(ext/pybind11)
 # Include fmt library
 # =============================================================================
 message(STATUS "INCLUDING FMT")
-add_subdirectory(${NMODL_PROJECT_SOURCE_DIR}/ext/fmt)
+add_subdirectory(${NMODL_PROJECT_SOURCE_DIR}/ext/fmt EXCLUDE_FROM_ALL)
+set_property(TARGET fmt PROPERTY POSITION_INDEPENDENT_CODE ON)
```

Comment:
Just a note - I think there was a question why PIC objects were needed - IIRC, this was needed for building pybind based nmodl module.

Reply:
Exactly, in the meantime I saw this too and it is really needed for pybind. Maybe this could be reworked in the future, but not worth to touch right now.